ios/FluentUI/Notification/FluentNotification.swift

```diff
@@ -108,6 +108,7 @@ public struct FluentNotification: View, ConfigurableTokenizedControl {
                                height: imageSize.height,
                                alignment: .center)
                         .foregroundColor(Color(dynamicColor: tokens.imageColor))
+                        .accessibilityHidden(true)
```

Comment:
we can't use https://developer.apple.com/documentation/swiftui/form/accessibilityelement(children:) .contain or combine to make the notificationview more consistent container behavior for voice over users? it should act somewhat like a dialog right?

Reply:
That'll put the scope of the work out of this PR, so I'll revert this for now and come back with a more VoiceOver focused PR.